cmd/grpc.go

```diff
@@ -48,13 +47,9 @@ func NewGrpcClient(log *slog.Logger, address string, cert, key, caCert []byte) (
 	dialOpts := []grpc.DialOption{
 		grpc.WithKeepaliveParams(kacp),
 		grpc.WithTransportCredentials(credentials.NewTLS(tlsConfig)),
-		grpc.WithBlock(),
 	}
 
-	ctx, cancel := context.WithTimeout(context.Background(), 15*time.Second)
-	defer cancel()
-
-	conn, err := grpc.DialContext(ctx, address, dialOpts...)
+	conn, err := grpc.NewClient(address, dialOpts...)
```

Comment:
What is the new behavior without blocking and without any timeout?

Reply:
You commented the same in this PR already: https://github.com/metal-stack/duros-go/pull/35

Reply:
May it hang forever without a timeout?

Reply:
No it will not block, but subsequent RPC Calls will error out if no connection was established yet